AdyenActions/Components/Await/AwaitComponent.swift

```diff
@@ -7,13 +7,18 @@
 import Adyen
 import Foundation
 
+///
+internal protocol AnyAwaitableAction {
+    var paymentData: String { get }
+}
+
 /// A component that handles Await action's.
 internal protocol AnyAwaitActionHandler: ActionComponent, Cancellable {
-    func handle(_ action: AwaitAction)
+    func handle(_ action: AnyAwaitableAction)
```

Comment:
Should `AnyAwaitableAction` be inherited from AwaitAction ?

Reply:
Not really. `AnyAwaitableAction` was created so we could use `QRCodeAction` in the `AwaitableComponent`